src/items/coherence.md

```diff
@@ -0,0 +1,79 @@
+## What is coherence and why do we care?
+
+Coherence means that for any given trait and type, there is one specific 
```

Comment:
Can you try to remove the motivational stuff from this text? The reference's only aim is to give a definition / specification of behavior, not explain why things matter and why we care.